### ISSUE TITLE
confused about loss (hypes) hyperparameters of the model 

### ISSUE THREAD

**Issue by jiankang1991** [MarvinTeichmann/tensorflow-fcn#3]
I am confused about what is `hypes` in loss. Where does this come from?
Thank you very much.
